### PR TITLE
Add sound to button and alarm sound

### PR DESCRIPTION
**Pull request by yohanaff**

### CODE REVIEW

**Review by francielebueno — APPROVED** [review]

File: index.html
Comment (TThaz) [inline index.html:1]: are you sure about this?

File: js/app.js
Comment (TThaz) [inline js/app.js:1]: This doesn't make much sense

**Review by TThaz — APPROVED** [review]
yeah, yeah, right

File: js/app.js
Comment (yohanaff) [reply js/app.js:1]: yeah, every Pomodoro has a alarm sound! :) Doesn't make sense if it doesn't have an alarm

**Pull request merged**